learn.rst

```diff
@@ -12,22 +12,22 @@ Learn about WSGI
   Titus Brown
 
 * `A Do-It-Yourself Framework
-  <http://pythonpaste.org/do-it-yourself-framework.html>`_ by Ian
+  <https://ianbicking.org/2008/04/17/webob-do-it-yourself-framework/index.html>`_ by Ian
```

Comment:
Sadly doesn't seem useful, it is just two dead links to pythonpaste.org.

Maybe https://paste.readthedocs.io/en/latest/do-it-yourself-framework.html would be a better target? Seems to be the new location for the original article.

Reply:
Yes..
Let me change it!

Reply:
Updated! Please check.
Thanks :)